### DISCUSSION TITLE
feature: Editor asset hot-reload

### DISCUSSION THREAD

**Discussion by buffotheclown** [AscensionGameDev/Intersect-Engine#2708] [Ideas]
**Is your feature request related to a problem? Please describe.**
Right now, I need to restart the editor for every single pixel fix I do to images while mapping and eventing.

**Describe the solution you'd like**
The Editor would be much user friendly if you could refresh, with a button press like f5, any new image files you replace or place in the appropriate folder during the editing process.

**Comment by WeylonSantana** [comment]
Maybe update the value of globals varables too in variable editor.

**Comment by Arufonsu** [comment]
@WeylonSantana we might be able to cover this one after/during the black magic happening behind scenes right now since we won't depend on win-forms anymore.

**Comment by WeylonSantana** [comment]
Agreed with you, I'm excited about it
